index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap"
+        rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <a href="" class="logo">My Logo</a>
+        <nav class="navigation">
+            <img src="images/Icon-Menu.png" alt="menu_icon" id="icon-menu" class="hamburger">
+        </nav>
+    </header>
+    <main>
+        <section class="hero-section">
+
+            <h1 id="hello" class="container">Hey there. <br>
+                I’m Mishelle.</h1>
+            <h2 id="developer" class="container">I’m a software <br> developer</h2>
+            <p> I can help you to build a product,<br> feature or website Look through <br> some of my work and experience!
+                <br> If you like what you see and have <br> a project you need coded, <br>
+                don’t hesitate to contact me.
+            </p>
+            <div class="socialMedia">
+                <ul class="social">
+                    <li><img src="images/twitter.png" alt="Twitter logo" class="social"><a href=""></a></li>
+                    <li><img src="images/Linkedin.png" alt="Linkedin logo" class="social"><a href=""></a></li>
+                    <li><img src="images/medium.png" alt="Medium" class="social"><a href=""></a></li>
+                    <li><img src="images/github.png" alt="GitHub" class="social"><a href=""></a></li>
+                    <li><img src="images/angellist.png" alt="Angellist" class="social"><a href=""></a></li>
+                </ul>
```

Comment:
- [x] Well done :clap: your project is very nice :+1: but there are some issues in your project first of all please add the `a` tag correctly you did add the `a` tag correction so kindly fix it :+1: check below :point_down: my suggestions :slightly_smiling_face: 

```suggestion
                <ul class="social">
                    <li><a href="#"><img src="images/twitter.png" alt="Twitter logo" class="social"></a></li>
                    <li><a href="#"><img src="images/Linkedin.png" alt="Linkedin logo" class="social"></a></li>
                    <li><a href="#"><img src="images/medium.png" alt="Medium" class="social"></a></li>
                    <li><a href="#"><img src="images/github.png" alt="GitHub" class="social"></a></li>
                    <li><a href="#"><img src="images/angellist.png" alt="Angellist" class="social"></a></li>
                </ul>
```